ZenPacks/zenoss/ZenPackLib/lib/functions.py

```diff
@@ -24,6 +24,18 @@
 
 # Private Functions #########################################################
 
+def object_from_string(name, text):
```

Comment:
I can't find object_from_string or add_class_text_method being used, nor can I find any of the new string templates being used. Am I missing something, or is this not completely implemented yet?

I'm not sure I like the idea of creating code this way. It makes debugging even harder, and it makes editors unable to help with code issues.